File: Introduction to programming/InputPrint.py
# Write a program in c to input and print details of an employee the id,name,salary using structure.

employee = []

def inputData():
    emp = {}
    emp["Id"] = int(input("Enter ID: "))
    emp["Name"] = input("Enter name : ")
    emp["Salary"] = int(input("Enter salary : "))

    employee.append(emp)
    
n = int(input("How many employee data you want to add? :"))

for _ in range(n):
    inputData()

for i in range(n):
    print(employee[i])

File: Introduction to programming/InputPrint.c
#include<stdio.h>

struct Employee {
    int ID;
    char Name[50];
    int Salary; 
};

int main() {
    int n;
    struct Employee emp[10];
    printf("How many employee data you want to add ? : ");
    scanf("%d", &n);

    for(int i = 0; i < n; i++){
        printf("\nEnter ID: ");
        scanf("%d", &emp[i].ID);

        printf("Enter Name: ");
        scanf("%s", emp[i].Name);

        printf("Enter Salary: ");
        scanf("%d", &emp[i].Salary);
    }

    printf("\nEmployee Details:\n");
    for(int i = 0; i < n; i++) {
        printf("ID : %d, Name : %s, Salary : %d\n",emp[i].ID, emp[i].Name, emp[i].Salary);
    }
    return 0;
}